src/rpc/names.cpp

```diff
@@ -483,7 +499,15 @@ name_show (const JSONRPCRequest& request)
 
   MaybeWalletForRequest wallet(request);
   LOCK2 (wallet.getLock (), cs_main);
-  return getNameInfo (options, name, data, wallet);
+  UniValue name_object = getNameInfo(options, name, data, wallet);
+  assert(!name_object["expired"].isNull());
+  const bool is_expired = name_object["expired"].get_bool();
+  if (is_expired && !allow_expired) {
```

Comment:
Please use GNU style matching the rest of the file here for the `{}`:

    if (something)
      {
        ...
      }